tests/data/results/www.jamieoliver.com/recipes-eggs-recipes-simple-cheese-omelette.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+    'author' => 'Jamie Oliver',
+    'categories' => [
+        'Breakfast',
+        'Jamie\'s Food Revolution',
+        'Eggs',
+        'Brilliant breakfast recipes',
+        'Family one-pan recipes',
+        'Mains',
+    ],
+    'cookingMethod' => '',
```

Comment:
Missing values should be null instead of an empty string. Same goes for the fish fingers recipe